custom/conf/app.ini.sample

```diff
@@ -336,8 +340,8 @@ ENABLE_CAPTCHA = false
 CAPTCHA_TYPE = image
 ; Enable recaptcha to use Google's recaptcha service
 ; Go to https://www.google.com/recaptcha/admin to sign up for a key
-RECAPTCHA_SECRET  =
-RECAPTCHA_SITEKEY =
+RECAPTCHA_SECRET  = 
```

Comment:
Why did you change this?

Reply:
Looks like a few spaces added there.. Will revert